index.js

```diff
@@ -78,9 +78,12 @@ module.exports = function (log, isReady) {
         log.since.once(function (since) {
           if(upto > since) {
             sv.destroy(function () { build(-1) })
-          } else
+          } else {
+            var opts = {gt: upto, live: true, seqs: true, values: true}
+            if (upto == -1)
+              opts.cache = false
```

Comment:
Side note:  I think `cache: Boolean` is an undocumented feature ? 
- https://github.com/flumedb/flumedb/#flumelogstreamopts
- https://github.com/flumedb/flumelog-offset ?? 
 
following trail it looks like it comes from (in flumelog-offset at least):
- https://github.com/flumedb/flumelog-offset/blob/master/inject.js#L56-L68
- https://github.com/dominictarr/pull-cursor#createstreamopts--source

Reply:
It is a little bit of a maze (welcome to Westworld!) but that is documented in https://github.com/dominictarr/pull-cursor